### ISSUE TITLE
Can't disable SC1114

### ISSUE THREAD

**Issue by mmazurkevich** [koalaman/shellcheck#1670]
Hi, 
I found that can't disable SC1114 inspection. The version of my shellcheck: 0.6.0 and this behavior also appears on [shellcheck.net](https://www.shellcheck.net/).

#### Here's a snippet or screenshot that shows the problem:

```sh
# shellcheck disable=SC1114
  #!/bin/sh
  echo "hi!"

```

#### Here's what shellcheck currently says:
```sh 
In example.sh line 2:
  #!/bin/sh
^-- SC1114: Remove leading spaces before the shebang.
^-- SC1128: The shebang must be on the first line. Delete blanks and move comments.

For more information:
  https://www.shellcheck.net/wiki/SC1114 -- Remove leading spaces before the ...
  https://www.shellcheck.net/wiki/SC1128 -- The shebang must be on the first ...
```

#### Here's what I wanted or expected to see:
Shellcheck allow disabling this inspection
